tagstudio/src/qt/helpers/text_wrapper.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2024 Travis Abendshien (CyanVoxel).
+# Licensed under the GPL-3.0 License.
+# Created for TagStudio: https://github.com/CyanVoxel/TagStudio
+
+from PIL import Image, ImageDraw, ImageFont
+
+
+def wrap_line(  # type: ignore
+    text: str,
+    font: ImageFont.ImageFont,
+    width: int = 256,
+    draw: ImageDraw.ImageDraw = None,
+) -> int:
+    """
+    Takes in a single line and returns the index it should be broken up at but
+    it only splits one Time
+    """
+    if draw is None:
+        bg = Image.new("RGB", (width, width), color="#1e1e1e")
+        draw = ImageDraw.Draw(bg)
+    if draw.textlength(text, font=font) > 256:
+        # print(draw.textlength(text, font=font))
+        for i in range(
+            int(len(text) / int(draw.textlength(text, font=font)) * 256) - 2,
+            0,
+            -1,
+        ):
+            if draw.textlength(text[:i], font=font) < 256:
+                # print(len(text))
+                # print(i)
```

Comment:
Change to use `width` variable instead of hardcoded values (plus remove print comments)
```suggestion
    if draw.textlength(text, font=font) > width:
        for i in range(
            int(len(text) / int(draw.textlength(text, font=font)) * width) - 2,
            0,
            -1,
        ):
            if draw.textlength(text[:i], font=font) < width:
                return i
```